.bazelrc

```diff
@@ -34,6 +34,9 @@ build:devcontainer --define=folly_so=1
 #  Bazel test runtime default: PATH=/bin:/usr/bin:/usr/local/bin
 #  Some python tests require access to /usr/sbin binaries (e.g. route, ifconfig)
 build --test_env=PATH=/bin:/usr/bin:/usr/local/bin:/usr/sbin
+#  Use MAGMA_ROOT from the host system in tests.
+#  Needed by python tests (e.g. freedomfi_one_tests in enodebd)
```

Comment:
:'( which test is this? This seems like something that is good to not make a precedent of

Reply:
> which test is this?
See line 38 - freedomfi_one_tests. It is used in the test to mock from the production config dir `/etc/config` to a directory in the magma repository. We can have a look if this can be refactored. Spontaneously I do not have a good idea to make this work with bazel and make. - worst case: it can be refactored when bazel is the only build system.

Reply:
sounds good to me!